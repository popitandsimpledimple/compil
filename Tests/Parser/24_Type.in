program main;
var
    e: double;
    x: integer;
    y: string;
    z : char;
begin
end.