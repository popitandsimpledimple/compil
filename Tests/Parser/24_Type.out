program
   main
   decls
      vars
         var
            e
            type
               double
            no expression
         var
            x
            type
               integer
            no expression
         var
            y
            type
               string
            no expression
         var
            z
            type
               char
            no expression
   block
      compounds statements
